src/StartMenu.ts

```diff
@@ -296,64 +298,85 @@ export class AEntryPointList implements IEntryPointList {
   private updateList(data: INamedSet[]) {
     const that = this;
 
+    const headerText = this.desc.name === 'Genes'? 'Gene Sets' : `${this.desc.name.substr(0, this.desc.name.length - 1)} Panels`;
+
+    this.$node.select('.header.subheader').text(`Predefined ${headerText}`);
+    this.$node.append('ul').classed('namedSets', true);
+
+
+    const predefinedNamedSets = data.filter((datum) => !datum.creator);
```

Comment:
why using a different criteria than it will be used whether to show the delete button or not?